File: ffmpeg/src/main/java/com/coder/ffmpeg/annotation/ImageFormat.java
package com.coder.ffmpeg.annotation;

import androidx.annotation.StringDef;

/**
 * @author: AnJoiner
 * @datetime: 19-12-22
 */
@StringDef({
        ImageFormat.PNG,
        ImageFormat.JPG,
})
public @interface ImageFormat {
    String PNG = "png";
    String JPG = "jpg";
}

File: ffmpeg-lite/src/main/java/com/coder/ffmpeg/annotation/ImageFormat.kt
package com.coder.ffmpeg.annotation

import androidx.annotation.StringDef

/**
 * @author: AnJoiner
 * @datetime: 19-12-22
 */
@StringDef(ImageFormat.PNG,
        ImageFormat.JPG)
annotation class ImageFormat {
    companion object {
        const val PNG = "png"
        const val JPG = "jpg"
    }
}